### PR TITLE
Replace tldp.org with other resources per #171

### PR DESCRIPTION
**Pull request by rpdelaney**
In #171 we discussed some reasons why tldp.org is not a reliable resource for shell scripting beginners. This PR replaces tldp.org in the `RESOURCES.md` for bash with much better guides and FAQs that will put newbies on the right track.

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/bash/blob/master/POLICIES.md)

### CODE REVIEW

File: docs/RESOURCES.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
-- [Bash man page](http://linux.die.net/man/1/bash)  
-- [The Linux Documentation Project Guides](http://tldp.org/guides.html)  
-- [Shellcheck provides shell script guidelines and help](http://www.shellcheck.net/)  
+- [Bash man page](http://linux.die.net/man/1/bash)
```
Comment (geirha) [inline docs/RESOURCES.md]: That manual at linux.die.net looks rather outdated. I'd remove it. The [GNU Bash Manual](http://gnu.org/s/bash/manual) covers everything the man-page does anyway, and the user can just run `man bash` on their own system to get the manual page corresponding to their installed version.

Comment (rpdelaney) [reply docs/RESOURCES.md]: This makes sense to me. Anybody who has bash installed has a bash manpage at their fingertips, so why link to one that might not correspond to what they have?

@budmc29 what do you think?

Comment (kotp) [reply docs/RESOURCES.md]: I will pretty often refer to the local tools such as man or a languages CLI tool to refer to documentation, so I wholeheartedly agree in referring to these.  No everyone has 24/7 access to the internet and having access regardless is a great advantage.

Comment (budmc29) [reply docs/RESOURCES.md]: @geirha, @rpdelaney the funny thing is that my Mac's bash is the 2005 version (can't check the version on my linux machine, but I never had to update bash in the past, so I assume it's the same).

I'd replace the link to the online man page with an instruction on how to access the local bash man page (I know it's sounds easy, but we need to keep in mind that there are people starting this track not knowing what a man page is).

Comment (budmc29) [reply docs/RESOURCES.md]: @kotp I'm curios what version is your local man page?

Comment (kotp) [reply docs/RESOURCES.md]: `GNU Bash 4.3`

When I ask for the version I get:

```plain
± % bash --version                                                       !10055
GNU bash, version 4.3.11(1)-release (x86_64-pc-linux-gnu)
Copyright (C) 2013 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
----- %< -----
```

Comment (rpdelaney) [reply docs/RESOURCES.md]: > I'd replace the link to the online man page with an instruction on how to access the local bash man page

Depending on how much detail you want this to have, this is going to be rather platform dependent. Do you intend to assume that everyone is using OSX? Or would just telling people "enter `man bash` in a terminal" be enough?

Comment (budmc29) [reply docs/RESOURCES.md]: You can do whatever you like, but of course we don't want to single out only an OS, so something general should be enough.

Comment (rpdelaney) [reply docs/RESOURCES.md]: 5a4c659

**Comment by budmc29** [comment]
Looks good, thank you for your contribution.

**Pull request merged**